runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -992,6 +993,34 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadSpaceToBatchND(const Operator
   subg.addOperation(std::move(new_op));
 }
 
+template <typename LoaderDomain, typename SpecificLoader>
+void BaseLoader<LoaderDomain, SpecificLoader>::loadBatchMatMul(const Operator *op, ir::Graph &subg)
+{
+  ir::OperandIndexSequence inputs;
+  ir::OperandIndexSequence outputs;
+
+  loadOperationIO(op, inputs, outputs);
+  ir::operation::BatchMatMul::Param param;
+
+  if (op->custom_options() == nullptr)
+  {
+    param.adj_x = false;
+    param.adj_y = false;
+  }
+  else
+  {
+    size_t custom_op_data_size = op->custom_options()->size();
+    auto custom_op_data = op->custom_options()->Data();
+    auto data_root = flexbuffers::GetRoot(custom_op_data, custom_op_data_size);
+    auto attr_map = data_root.AsMap();
+    param.adj_x = attr_map["adj_x"].AsBool();
```

Comment:
@hseok-oh How this map works when "adj_x" field does not exist? Does it throw exception?